apps/ae_socket_connector/lib/client_runner.ex

```diff
@@ -38,11 +38,10 @@ defmodule ClientRunner do
 
               auto_approval
             end,
-            channels_update: fn round_initiator, nonce ->
+            channels_update: fn round_initiator, nonce, method ->
               Logger.debug(
                 "callback received round is: #{inspect(nonce)} round_initiator is: #{
-                  inspect(round_initiator)
-                }}",
+                  inspect(round_initiator)} method is #{inspect method}}",
```

Comment:
With or without `()` I don't care too much, just be consistent.

Reply:
fixed